pipeline/utils/notifications_test.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from pipeline.types import NotificationInfo, NotificationType
+from pipeline.utils.notifications import notifications_per_user
+
+
+def test_return_lst_1_notification():
+    notifications = [
+        NotificationInfo(
+            recipient="A",
```

Comment:
The reason you got stuck on the recipient issue is because the type of this property is `User`, not a string or user ID like you have used before in other classes.

Check out the type definition:

https://github.com/scarletstudio/butterfly/blob/bf74db6426ead1e0fbd13d7254c388d56c091d84/pipeline/types/notification.py#L27-L34

Instead, you can create a recipient like this (make share to import the `User` type):

```python
notifications = [
  NotificationInfo(
    recipient=User(uid="A"),
    notification_type=NotificationType.CHECK_MATCH
  )
]
```

You can also leave out `messagers` since it is not affected by the function you are writing, but if you were setting that property, it would have to be a list of strings, per the type definition.